src/hardware/serialport/serialdummy.cpp

```diff
@@ -23,7 +23,9 @@
 #include "serialdummy.h"
 #include "serialport.h"
 
-CSerialDummy::CSerialDummy(Bitu id,	CommandLine* cmd):CSerial(id, cmd) {
+CSerialDummy::CSerialDummy(const uint8_t port_idx, CommandLine *cmd)
```

Comment:
Changes in this file are 1:1 type replacements, reformatting, and the `(void)unused;`, which we might change.